### PR TITLE
Add Hide UI option during gameplay

### PR DESCRIPTION
**Pull request by giang428**
adding an option to hide score text, combo text, accuracy text and health bar during gameplay

### CODE REVIEW

**Comment by Rian8337** [comment]
There are so many unnecessary changes here. Please only change/add what is needed to achieve what you want.

And in best practice, you should name your commit properly so that it is easier to identify what your commit does without having to inspect the changes.

File: AndEngine/build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:4.1.2'
```
Comment (Rian8337) [inline AndEngine/build.gradle:9]: Are these changes to gradle necessary?

File: src/ru/nsu/ccfit/zuev/osu/game/GameScene.java [hunk 2]
```diff
@@ -1241,7 +1241,13 @@ else if (stat.getMod().contains(GameMod.MOD_PERFECT)) {
...
             addPassiveObject(scoreBoard);
         }
-
+        if(Config.isHideUI()){
+            scorebar.setVisible(false);
+            comboText.detachFromScene();
+            scoreText.detachFromScene();
+            accText.detachFromScene();
+            scoreShadow.detachFromScene();
+        }
```
Comment (Rian8337) [inline src/ru/nsu/ccfit/zuev/osu/game/GameScene.java:1250]: I think it is better if you check this when the game attaches these scenes for the first time, found here (`scorebar` needs to be done separately though).
https://github.com/osudroid/osu-droid/blob/2b192612bbf80557422ed9413e7e2519ed14aa50/src/ru/nsu/ccfit/zuev/osu/game/GameScene.java#L934-L942

And if you are not aware, `ScoreBar` would appear again if a break section in the beatmap exists (once the break section is over).
https://github.com/osudroid/osu-droid/blob/2b192612bbf80557422ed9413e7e2519ed14aa50/src/ru/nsu/ccfit/zuev/osu/game/GameScene.java#L1447-L1453

**Review by Rian8337 — CHANGES_REQUESTED** [review]

**Comment by Ycro-osu-catch** [comment]
During Gameplay
In-Game Interface On / Off
as in osu desktop / pc?

**Comment by Rian8337** [comment]
I see you have committed the fix in your master branch (which I don't actually recommend, but oh well). You can either create a new pull request with the branch or continue to work on this pull request.

**Pull request closed without merging**